examples/pa_engine_multiple_unit_example.py

```diff
@@ -49,15 +49,16 @@ def main():
         startdate = "20180101"
         enddate = "20181231"
         frequency = "Monthly"
+        holdings = "B&H"
```

Comment:
Is this better to make the variable name too as `holdingsmode`? Also, it may be good to have this variable at line 49 as it is related to accounts and benchmarks.